geoh5py/shared/concatenation/object.py

```diff
@@ -148,9 +149,13 @@ def property_groups(self) -> list | None:
                 property_groups = []
 
             for key in property_groups:
-                self.fetch_property_group(
-                    **self.concatenator.get_concatenated_attributes(key), on_file=True
-                )
+                attributes = {
+                    INV_KEY_MAP.get(key, key): val
```

Comment:
question: Why this option could'nt be implemented into the get_concatenated_attributes function?
Like inv_key_map: dict | None, If isinstance(inv_key_map, dict) => convert

Reply:
I tried, but get concatenated_attribute gets called in other places and it started to be an heavy change. This is bare minimum I needed to get rid of map_keys in the property_group instantiation.